### DISCUSSION TITLE
ESLint React Errors

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [neonexus/sails-react-bootstrap-webpack#68] [Q&A] [answered]
Hi @neonexus ,

_Using [ESLint React](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/forbid-prop-types.md)_

`Prop type "object" is forbidden  react/forbid-prop-types` is the common error when declaring `PropTypes` for components like:

```
Component.propTypes = {
    api: PropTypes.object.isRequired,
};
```
Same with the error `Must use destructuring state assignmenteslint[react/destructuring-assignment]` when accessing props.

I'm sure you already know about this. Is there any specific reason that you ignored these errors?

Thanks.

**Comment by neonexus** [comment, marked as answer]
@MaheshkumarSundaram it sounds like the version of ESLint you are running isn't using the `.eslintrc` config in the repo, as I have both `react/forbid-prop-types` & `react/destructuring-assignment` turned off. 

I use `npm run lint`, which will use the version the repo requires in `package.json`. It's also run when `npm run test` is used. I use [Travis CI](https://travis-ci.com) (can see [the build history here](https://app.travis-ci.com/github/neonexus/sails-react-bootstrap-webpack)) to help make sure I don't break anything.

**Comment by neonexus** [comment]
Re-opening this discussion for ease of viewing.
